### PR TITLE
London Class 7 - Hozan Ali - Git/GitHub - Week 2

### PR DESCRIPTION
**Pull request by Hozan94**
fixed the erros in some of the questions

# Your Details

- Your Name: Hozan Ali
- Your City: London
- Your Slack Name: Hozan Ali

# Homework Details

- Module: Git/GitHub
- Week: 2

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,77 +4,80 @@
...
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website look nicer
```
Comment (ybayraktutan) [inline homework.md:32]: It can be more descriptive.

File: homework.md [hunk 2]
```diff
@@ -4,77 +4,80 @@
...
```
Comment (ybayraktutan) [inline homework.md:31]: Can be much descriptive.

**Comment by ybayraktutan** [comment]
Great answers - well done! One small thing you can improve on Q5. Great job!.

File: homework.md [hunk 3]
```diff
@@ -92,13 +95,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send your code to Github (bring the changes back to GitHub)
```
Comment (iTrip23) [inline homework.md:98]: Hi Hozan,
nice work so far, you need to think a bit more for this one, you are confusing commit with push.

File: homework.md [hunk 4]
```diff
@@ -4,77 +4,80 @@
...
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```
Comment (iTrip23) [inline homework.md:17]: Hi Hozan,
You have a typo here on question 3, 
2 * 3 * 6 ... not 2 _ 3 _ 6 ...

**Comment by iTrip23** [comment]
Hi Hozan,
good answers overall, have another look at questions 3 & 13, I could be wrong, I had also question 14 in my file, don't know what happened there....
Nice work

File: homework.md (hunk 2 above)
Comment (Hozan94) [reply homework.md:31]: Thanks for the feedback, will look into that.

File: homework.md (hunk 3 above)
Comment (Hozan94) [reply homework.md:98]: Hi iTrip23, 

I think the answer might be right beacuse we when we commit, we send changes back to GitHub.

And push is used to send the these commits to the remote repo.

maybe @ChrisOwen101 can correct me if I am wrong.

File: homework.md (hunk 4 above)
Comment (Hozan94) [reply homework.md:17]: sure things, thanks for letting me knwo.

File: homework.md (hunk 2 above)
Comment (Hozan94) [reply homework.md:31]: thanks for the feedback, will look into that.

File: homework.md [hunk 5]
```diff
@@ -4,77 +4,80 @@
...
 
-It send it to a friend
+The repository is copied to your local machine with the help of Git.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Asks the repository owner to merge the changes with the original repository
```
Comment (louisechow) [inline homework.md:75]: There can also be a case where you are the repository owner and you want to send a Pull Request to your team members. 

@Hozan94 in the case I just described, what is the purpose of the pull request?

File: homework.md (hunk 3 above)
Comment (louisechow) [reply homework.md:98]: @Hozan94 Github is the remote repository so `push` is the command to send changes to the Github.

So, what do you think `commit` does then?

**Comment by louisechow** [comment]
Great job! 🌟

I added some comments onto the questions about `Pull Request` and `commit`. See if you can answer those and reach out if you want to discuss those topics more!

File: homework.md [hunk 6]
```diff
@@ -4,77 +4,80 @@
...
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the website to make it interactive
```
Comment (louisechow) [inline homework.md:46]: Keep in mind that Javascript can also be used on the backend too so it's not just used for website interaction. You will be learning about Node later in the course.

File: homework.md (hunk 5 above)
Comment (Hozan94) [reply homework.md:75]: @louisechow So in this case, the purpose of "Pull Request" is to tell collaborators that a change has been `pushed` (or merged ?) to the original repository.

File: homework.md (hunk 3 above)
Comment (Hozan94) [reply homework.md:98]: @louisechow then `commit` send changes to the local repository (the repo that is stored on your local computer ? ).

Also if that is the case then how come you have the option to make `commit` on GitHub Web, even though you have not cloned the repo?

**Pull request closed without merging**